.changes/v3.4.0/716-bug-fixes.md

```diff
@@ -0,0 +1 @@
+* Primary NIC for `vcd_vapp_vm` and `vcd_vapp` is done in cold fashion [GH-716]
```

Comment:
There's a word "removal" missing.

BTW, it sounds as if we are powering off the VM by our selves. Is this the case?

Reply:
> There's a word "removal" missing.

Oops. Fixed.



> BTW, it sounds as if we are powering off the VM by our selves. Is this the case?

Yes. That has always been the case for cold operations. And before we introduced support for hot operations (cpu, memory, nic) it was always the case.